pkg/cmd/project/item-delete/item_delete.go

```diff
@@ -126,6 +126,8 @@ func printResults(config deleteItemConfig) error {
 }
 
 func printJSON(config deleteItemConfig, ID githubv4.ID) error {
```

Comment:
```suggestion
func printJSON(config deleteItemConfig, id githubv4.ID) error {
```

Small nit here, I know this was existing code, but since we are already here it makes sense to fix this.